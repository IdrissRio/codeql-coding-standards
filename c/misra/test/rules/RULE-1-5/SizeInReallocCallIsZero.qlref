rules/RULE-1-5/SizeInReallocCallIsZero.ql