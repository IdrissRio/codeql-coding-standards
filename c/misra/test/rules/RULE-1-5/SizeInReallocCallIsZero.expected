| test.c:14:3:14:9 | call to realloc | Size argument '$@' may equal zero in realloc call, resulting in obsolescent and/or implementation-defined behavior. | test.c:14:14:14:14 | 0 | 0 |
